static/app/components/events/eventStatisticalDetector/aggregateSpanOps/pieChart.tsx

```diff
@@ -95,106 +104,190 @@ class PieChart extends Component<Props> {
+                    {op}
+                  </span>
+                  <Tooltip
+                    skipWrapper
+                    title={t(
+                      `Total time for %s went %s from %s to %s`,
+                      op,
+                      percentageText,
+                      oldValue,
+                      newValue
+                    )}
+                  >
+                    <SpanOpChange regressed>
+                      {percentageText} {percentage}
+                    </SpanOpChange>
+                  </Tooltip>
+                </StyledLegendWrapper>
+              );
+            })}
+        </LegendWrapper>
+        <BaseChart
+          ref={this.chart}
+          colors={this.pieChartSliceColors}
+          // when legend highlights it does NOT pass dataIndex :(
+          onHighlight={({name}) => {
+            if (
+              !this.isInitialSelected ||
+              !name ||
+              firstSeries.data[this.selected].name === name
+            ) {
+              return;
+            }
+
+            // Unhighlight if not initial "highlight" event and
+            // if name exists (i.e. not dispatched from cDM) and
+            // highlighted series name is different than the initially selected series name
+            this.downplay(this.selected);
+            this.isInitialSelected = false;
+          }}
+          onMouseOver={({dataIndex}) => {
+            if (!this.isInitialSelected) {
+              return;
+            }
+            if (dataIndex === this.selected) {
+              return;
+            }
+            this.downplay(this.selected);
+            this.isInitialSelected = false;
+          }}
+          {...props}
+          tooltip={{
+            formatter: data => {
+              return [
+                '<div class="tooltip-series">',
+                `<div><span class="tooltip-label">${data.marker}<strong>${data.name}</strong></span></div>`,
+                '</div>',
+                `<div class="tooltip-footer">${getDuration(
+                  this.props.data[data.name].oldBaseline / 1000,
+                  2,
+                  true
+                )} to ${getDuration(
+                  this.props.data[data.name].newBaseline / 1000,
+                  2,
+                  true
+                )}</div>`,
+                '</div>',
+                '<div class="tooltip-arrow"></div>',
+              ].join('');
             },
-            emphasis: {
+          }}
+          series={[
+            PieSeries({
+              name: firstSeries.seriesName,
+              data: firstSeries.data,
+              avoidLabelOverlap: false,
               label: {
+                position: 'inside',
+                formatter: params => {
+                  return `${Math.round(Number(params.percent))}%`;
+                },
                 show: true,
+                color: theme.background,
+                fontSize: 12,
+                fontWeight: 600,
```

Comment:
```suggestion
                formatter: params => {
                  return `${params.name} ${Math.round(Number(params.percent))}%`;
                },
                show: true,
                color: theme.background,
                overflow: 'break',
                width: 40
```

<img width="332" alt="Screenshot 2023-10-20 at 4 34 20 PM" src="https://github.com/getsentry/sentry/assets/22846452/47d8ce3e-6003-4b09-892b-442c8873e384">

I don't think the font-weight is adding too much here, it looks crisper without it. These other settings let us show the param name and break it up into two lines too. It's a bit of a hack but it makes the segments clearer without the hovering